ports/geckolib/properties.mako.rs

```diff
@@ -610,6 +610,23 @@ fn static_assert() {
 
 </%self:impl_trait>
 
+<%self:impl_trait style_struct_name="Color"
+                  skip_longhands="color"
```

Comment:
Do we support skip_longhands="*"? If not, we should add it, and we should use it here.